doc/user/bgp.rst

```diff
@@ -35,6 +35,43 @@ be specified (:ref:`common-invocation-options`).
    of ``0.0.0.0`` / ``::``. This can be useful to constrain bgpd to an internal
    address, or to run multiple bgpd processes on one host.
 
+.. option:: -n, --no_kernel
+
+   Do not install learned routes into the linux kernel.  This option is useful
+   for a route-reflector environment or if you are running multiple bgp
+   processes in the same namespace.  This option is different than the --no_zebra
+   option in that a ZAPI connection is made.
+
+.. option:: -S, --skip_runas
+
+   Skip the normal process of checking capabilities and changing user and group
+   information.
+
+.. option:: -e, --ecmp
+
+   Run BGP with a limited ecmp capability, that is different than what BGP
+   was compiled with.  The value specified must be greater than 0 and less
+   than or equal to the MULTIPATH_NUM specified on compilation.
+
+.. option:: -I, --int_num
+
+   Set the instance number to be used for this bgp process.  This is useful
+   if you have multiple BGP processes connecting to the same zebra and intend
+   to install routes.
+
+.. option:: -Z, --no_zebra
+
+   Do not communicate with zebra at all.  This is different than the --no_kernel
+   option in that we do not even open a ZAPI connection to the zebra process.
+
+... option:: -s, --socket_size
+
+   When opening tcp connections to our peers, set the socket send buffer
```

Comment:
and here also same as in the previous commit, just a minor modification to reflect what is really changed (send and receive) :+1:

Reply:
fixed